pypeit/ginga.py

```diff
@@ -45,10 +53,17 @@ def connect_to_ginga(host='localhost', port=9000, raise_err=False):
     try:
         tmp = ginga.get_current_workspace()
     except:
+        if allow_new:
```

Comment:
good one.

Reply:
Thanks!  Was worried there was some reason not to do this.  Should we make the `allow_new=True` by default?